README.md

```diff
@@ -55,6 +55,8 @@ can be used from the build directory (without installing it system-wide).
 
 ### Support Matrix legend:
 
+:large_blue_diamond: Tested in CI extensively, passed the official upstream OpenCL-CTS
```

Comment:
Perhaps make it more emphasized that the port has been given the official conformance status?